private/examples/multiplatform_py_test/WORKSPACE.bazel

```diff
@@ -0,0 +1,45 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:local.bzl", "local_repository")
+
+http_archive(
```

Comment:
Is there still a `repositories.bzl`? Should use that to provide `rules_python`

Reply:
Restored